simulator/src/scrypto_bindgen/translation.rs

```diff
@@ -223,6 +224,20 @@ where
             "GlobalAddressReservation".to_owned()
         }
         TypeKind::Custom(ScryptoCustomTypeKind::Own) => match type_validation {
+            TypeValidation::Custom(ScryptoCustomTypeValidation::Own(
+                OwnValidation::IsTypedObject(package_address, bp_name),
+            )) if package_address == Some(RESOURCE_PACKAGE)
+                && bp_name == FUNGIBLE_BUCKET_BLUEPRINT =>
+            {
+                "FungibleBucket".to_owned()
```

Comment:
Nice! :). How did you spot this out of interest?

We should probably also handle `FungibleProof`/`NonFungibleProof`/`FungibleVault`/`NonFungibleVault`/`FungibleResourceManager`/`NonFungibleResourceManager` too?

Reply:
Thanks :)
I was checking where else `OwnValidation::IsBucket` enum variant is used and found this one.

I will add those missing ones in separate PR if that's ok.